docs/en/interfaces/formats.md

```diff
@@ -25,8 +25,8 @@ The supported formats are:
 | [Vertical](#vertical)                                                                   | ✗     | ✔      |
 | [VerticalRaw](#verticalraw)                                                             | ✗     | ✔      |
 | [JSON](#json)                                                                           | ✗     | ✔      |
-| [JSONAsString](#json)                                                                   | ✗     | ✗      |
-| [JSONString](#jsonasstring)                                                             | ✔     | ✔      |
+| [JSONAsString](#jsonasstring)                                                           | ✔     | ✗      |
+| [JSONString](#jsonstring)                                                               | ✗     | ✔      |
```

Comment:
JSONStrings maybe?

Reply:
It was there here before. I would need to check what's this (too many JSON options actually, very confusing)